src/screens/DiscoveryScreenV2/elements/DiscoveryItemList.js

```diff
@@ -191,14 +191,14 @@ const styles = StyleSheet.create({
     fontFamily: fonts.inter[600],
     fontWeight: 'bold',
     fontSize: 12,
-    color: COLORS.white2
+    color: COLORS.white
   },
   profilepicture: {
     width: 48,
     height: 48,
     borderRadius: 24,
     resizeMode: 'cover',
-    borderColor: COLORS.lightgrey,
+    borderColor: COLORS.gray100,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The border color for the profile picture has been updated from `COLORS.lightgrey` to `COLORS.gray100`. Make sure this change doesn't affect the overall design and visibility of the profile picture, especially on different backgrounds.
```diff
-    borderColor: COLORS.lightgrey,
+    borderColor: COLORS.gray100,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->